libmamba/src/core/environment.cpp

```diff
@@ -267,7 +269,49 @@ namespace mamba
                 throw std::runtime_error("HOME not set.");
             }
 #endif
-            return maybe_home;
+            return fs::u8path(maybe_home);
+        }
+
+        fs::u8path user_config_dir()
+        {
+            std::string maybe_user_config_dir = env::get("XDG_CONFIG_HOME").value_or("");
+            if (maybe_user_config_dir.empty())
+            {
+#ifdef _WIN32
+                maybe_user_config_dir = ::mamba::win::get_folder("roamingappdata");
```

Comment:
@pavelzw see discussion in the other PR. I'm fine with the decision to use this as the Windows default folder; people can always set the XDG vars.

Reply:
On Cygwin I'd be surprised if `~/.config` wasn't read by default though...

Reply:
I'm also fine with this. If people want to have XDG like stuff on cygwin, they should just set it in their bashrc/bash_profile.